doc/man3/RAND_DRBG_generate.pod

```diff
@@ -59,16 +61,18 @@ If necessary, they can be changed using L<RAND_DRBG_set_reseed_interval(3)>
 and L<RAND_DRBG_set_reseed_time_interval(3)>, respectively.
 
 A request for prediction resistance can only be satisfied by pulling fresh
-entropy from one of the approved entropy sources listed in section 5.5.2 of
-[NIST SP 800-90C].
-Since the default DRBG implementation does not have access to such an approved
-entropy source, a request for prediction resistance will always fail.
-In other words, prediction resistance is currently not supported yet by the DRBG.
+entropy from a live entropy source (section 5.5.2 of [NIST SP 800-90C]).
+Some random seed sources are not live entropy sources and a request
+for prediction resistance with such a source configured will always fail.
+For example, configuring with I<--with-rand-seed=none> will ensure that
+prediction resistance always fails.
```

Comment:
Can you rewrite that? Maybe it should say something that they need to make sure a live entropy source is configured.

Reply:
Done.

Reply:
Now fix that 3 more times?

Reply:
Two more fixed.  I can't locate a third.
Also made the text under the function descriptions for generate and reseed more consistent.